scripts/travis.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+npm run install:packages
+
+export CLIENT_ID=0oad5a19u0IVnKU1g0h7
+export REDIRECT_URI=http://localhost:8080/implicit/callback
+
+cd packages/jwt-verifier
+npm test > ./jwt-verifier-test.log
+
+if (cat ./jwt-verifier-test.log | grep -i -e "failed:" -e "failing" > /dev/null); then
+    echo "[ERROR] jwt-verifier tests failed!"
+    cat ./jwt-verifier-test.log
+    exit -1
+else
+    echo "jwt-verifier tests successful."
+fi
+
+export CLIENT_ID=0oad5983foWn2Gefr0h7
```

Comment:
Since these are shared, we should put them into a travis environment var.

Reply:
Actually, since we are testing both auth code and implicit flows, and even within these flows, some apps use port 8080, while some others 3000, we can't have just 1 client Id. Will need to set this env var before each test. 
I already have a CLIENT_ID travis env var, but that is for a web app.

Reply:
Don't we have control over these apps? We should be able to use the default (or fallback) callback URIs instead of hardcoding these. Additionally, I'd prefer to use environment vars if possible.